src/libraries/System.Runtime.InteropServices.JavaScript/src/System.Runtime.InteropServices.JavaScript.csproj

```diff
@@ -3,8 +3,6 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>enable</Nullable>
-    <!-- Use following lines to write the generated files to disk. -->
-    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <RootNamespace></RootNamespace>
```

Comment:
Why is the namespace unset? Please add a comment

Reply:
Otherwise that VS will create classes in `System.Runtime.InteropServices.JavaScript.System.Runtime.InteropServices.JavaScript` namespace, because they are in that folder. Do other projects have the same issue ? I guess it could be removed now.

Reply:
will do